src/cliffordsynthesis/CliffordSynthesizer.cpp

```diff
@@ -66,7 +69,7 @@ void CliffordSynthesizer::synthesize(const Configuration& config) {
     gateOptimalSynthesis(encoderConfig, lower, upper);
     break;
   case TargetMetric::Depth:
-    depthOptimalSynthesis(encoderConfig, lower, upper);
+    depthOptimalSynthesis(encoderConfig, config.minimalTimeSteps, upper);
```

Comment:
This should still be passed `lower`, right?

Reply:
Yes of course.